tests/suites/test_suite_psa_crypto.data

```diff
@@ -4709,6 +4717,98 @@ PSA key derivation: TLS 1.2 PRF SHA-256, derive key export, 1+41
 depends_on:PSA_WANT_ALG_SHA_256:PSA_WANT_ALG_TLS12_PRF
 derive_key_export:PSA_ALG_TLS12_PRF(PSA_ALG_SHA_256):"0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b":"000102030405060708090a0b0c":"f0f1f2f3f4f5f6f7f8f9":1:41
 
+PSA key derivation: HKDF-SHA-256 -> AES-128
+depends_on:PSA_WANT_ALG_HKDF:PSA_WANT_ALG_SHA_256:PSA_WANT_KEY_TYPE_AES
+derive_key_type:PSA_ALG_HKDF(PSA_ALG_SHA_256):"0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b0b":"000102030405060708090a0b0c":"f0f1f2f3f4f5f6f7f8f9":PSA_KEY_TYPE_AES:128:"3cb25f25faacd57a90434f64d0362f2a"
```

Comment:
Question: where do the `expected_export` values come from? I assume they were just generated with Mbed TLS?

Reply:
The expected results were generated using the python script provided by @gilles-peskine-arm in PR https://github.com/ARMmbed/mbedtls/pull/5218.

Reply:
It is mentioned in the commit message https://github.com/ARMmbed/mbedtls/pull/5139/commits/cb3eacd6913a721f348f73247fffbfa77ce2dd01.

Reply:
Oh sorry, I missed it, my fault.